code/game/objects/items/devices/scanners.dm

```diff
@@ -21,14 +21,19 @@ BREATH ANALYZER
 	matter = list(DEFAULT_WALL_MATERIAL = 200)
 	origin_tech = list(TECH_MAGNET = 1, TECH_BIO = 1)
 	var/mode = 1
+	var/last_scan = 0
 
 /obj/item/device/healthanalyzer/attack(mob/living/M, mob/living/user)
-	health_scan_mob(M, user, mode)
-	add_fingerprint(user)
+	if(last_scan <= world.time - 20) //Spam limiter.
+		last_scan = world.time
+		health_scan_mob(M, user, mode)
+		add_fingerprint(user)
```

Comment:
Don't limit the scanning itself, limit the playsound() instead.

Reply:
i playtested and it seems fine. prevents yourself from spamming the chat.

Reply:
fair enough!